ydb/core/viewer/json_acl.h

```diff
@@ -32,206 +33,271 @@ class TJsonACL : public TViewerPipeClient<TJsonACL> {
+          - name: path
+            in: query
+            description: schema path
+            required: true
+            type: string
+          - name: merge_rules
+            in: query
+            description: merge access rights into access rules
+            type: boolean
+          - name: timeout
+            in: query
+            description: timeout in ms
+            required: false
+            type: integer
+          responses:
+            200:
+              description: OK
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      Common:
+                        type: object
+                        properties:
+                          Path:
+                            type: string
+                          Owner:
+                            type: string
+                          ACL:
+                            type: array
+                            items:
+                              type: object
+                              properties:
+                                AccessType:
+                                  type: string
+                                Subject:
+                                  type: string
+                                AccessRules:
+                                  type: array
+                                  items:
+                                    type: string
+                                AccessRights:
+                                  type: array
+                                  items:
+                                    type: string
+                                InheritanceType:
+                                  type: array
+                                  items:
+                                    type: string
+                          EffectiveACL:
+                            type: array
+                            items:
+                              type: object
+                              properties:
+                                AccessType:
+                                  type: string
+                                Subject:
+                                  type: string
+                                AccessRules:
+                                  type: array
+                                  items:
+                                    type: string
+                                AccessRights:
+                                  type: array
+                                  items:
+                                    type: string
+                                InheritanceType:
+                                  type: array
+                                  items:
+                                    type: string
+            400:
+              description: Bad Request
+            403:
+              description: Forbidden
+            504:
+              description: Gateway Timeout
+
```

Comment:
it could return 307 Temporary Redirect, 500 Internal Error also

Reply:
there is no requirement to describe all of them